ui/src/features/Dialog/components/workspace/useWorkspace.tsx

```diff
@@ -0,0 +1,19 @@
+import { StarIcon } from "@radix-ui/react-icons";
```

Comment:
Please try to use phosphorus icons, not radix. Radix is very limited compared to phsphoros.

Reply:
In this case `Icon` isn't even used. Can we update the code it make it optional?

Reply:
Ah in that case maybe just leave this as is. I think the Dialog feature will be simplified so it doesn't have the tabular functionality, so these icons will be deleted too.
Once you read this you can resolve this convo 👍🏻

Reply:
I have updated the icon still. Resolving this!